Protocol.md

```diff
@@ -91,7 +91,6 @@ The following are features of Data and Metadata.
 The RSocket protocol uses a lower level transport protocol to carry RSocket frames. A transport protocol MUST provide the following:
 
 1. Unicast [Reliable Delivery](https://en.wikipedia.org/wiki/Reliability_(computer_networking)).
-1. [Connection-Oriented](https://en.wikipedia.org/wiki/Connection-oriented_communication) and preservation of frame ordering. Frame A sent before Frame B MUST arrive in source order. i.e. if Frame A is sent by the same source as Frame B, then Frame A will always arrive before Frame B. No assumptions about ordering across sources is assumed.
```

Comment:
Should this change to stream oriented? We allow reordering of streams, but each stream has the previous guarantees as seen by higher layers?

Reply:
Yes. That is correct. I tried to look for stream-oriented but there was no such a statement